src/App.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {
+  Switch,
+  Route,
+  HashRouter as Router,
+} from 'react-router-dom';
+import './style/app.css';
```

Comment:
- [ ] When I run your project locally, `npm start` throws the following error because this module is missing. Either delete this or add the module again.
```
Starting the development server...

Failed to compile.

./src/App.js
Module not found: Can't resolve './style/app.css' in 'C:\Users\PC\Desktop\TSE\sports-capstone\src'
```